### PR TITLE
Add the convenience function Header.is_empty

### PR DESCRIPTION
**Pull request by dinosaure**

### CODE REVIEW

File: cohttp/src/header.mli [hunk 1]
```diff
@@ -22,6 +22,9 @@ type t [@@deriving sexp]
 (** Construct a fresh, empty map of HTTP headers *)
 val init : unit -> t
 
+(** Test whether a map is empty or not. *)
```
Comment (rgrinberg) [inline cohttp/src/header.mli]: map? you mean header here

Comment (dinosaure) [inline cohttp/src/header.mli]: Done 👍

File: cohttp/src/header.mli [hunk 2]
```diff
@@ -22,6 +22,9 @@ type t [@@deriving sexp]
 (** Construct a fresh, empty map of HTTP headers *)
 val init : unit -> t
 
+(** Test whether a HTTP headers is empty or not. *)
+val is_empty : t -> unit
```
Comment (rgrinberg) [inline cohttp/src/header.mli]: `-> bool` I think

And it should `whether HTTP headers are empty...`

Comment (dinosaure) [inline cohttp/src/header.mli]: 😄

**Pull request merged**